### PR TITLE
Implemented export torrent endpoint

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

**Comment by devos50** [comment]
@lfdversluis can you review?

File: Tribler/Core/Modules/restapi/downloads_endpoint.py [hunk 1]
```diff
@@ -208,3 +209,37 @@ def render_PATCH(self, request):
...
+
+            **Example response**:
+
+            The contents of the .torrent file.
+        """
+        download = self.session.get_download(self.infohash)
+        if not download:
+            return DownloadExportTorrentEndpoint.return_404(request)
+
+        request.setHeader(b'content-type', 'application/x-bittorrent')
```
Comment (lfdversluis) [inline Tribler/Core/Modules/restapi/downloads_endpoint.py]: is it necessary to make 'content-type' and the other bytes?

**Comment by lfdversluis** [comment]
If my comment is invalid, I am happy :+1:

File: Tribler/Core/Modules/restapi/downloads_endpoint.py (hunk 1 above)
Comment (devos50) [inline Tribler/Core/Modules/restapi/downloads_endpoint.py]: It's necessary to make web browsers happy (so they download the file immediately) :)

**Comment by devos50** [comment]
@whirm ready for review

**Pull request merged**